src/Symfony/Component/Translation/Translatable.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation;
+
+/**
+ * @author Nate Wiebe <nate@northern.co>
+ */
+final class Translatable
+{
+    private $messageId;
+
+    private $parameters;
+
+    private $domain;
+
+    public function __construct(string $messageId, array $parameters = [], string $domain = 'messages')
+    {
+        $this->messageId = $messageId;
+        $this->parameters = $parameters;
+        $this->domain = $domain;
+    }
+
+    /**
+     * @return string The message id (may also be an object that can be cast to string)
+     */
+    public function getMessageId(): string
```

Comment:
I would use `getMessage()` as a message can be a plain English sentence or an id.

Reply:
@fabpot i proposed messageId. I think it's semantically more clear, as ultimately this _is_ the ID value (even if it's a plain English sentence).

Reply:
`message` is what is used everywhere in the component, we don't use message id anywhere, so I still think that message should be used here.

Reply:
I agree with this. Updated the PR.